.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+# Description
+
+<!--
+  A summary of the change, including any relevant motivation and context.
+
+  If relevant, include a description both of the existing behavior and of the new behavior.
+-->
+
+## Links
+
+<!--
+  Any relevant links to external resources; ie, specification documents, jira
+  items, related PRs, honeybadger errors, etc
+-->
+
+- [spec]()
+- [jira]()
+
+## Type of change
+
+- [ ] Bug fix <!-- non-breaking change which fixes an issue -->
+- [ ] New feature <!-- non-breaking change which adds functionality -->
+- [ ] Refactoring <!-- no functional changes -->
+- [ ] Breaking change <!-- fix or feature that would cause existing functionality to not work as expected -->
```

Comment:
Why is this even an option? :cry:

Reply:
hah, maybe this is just me being overly cautious and/or pessimistic.

On the one hand, breaking changes do happen and are sometimes even necessary, and I'd like to encourage honesty when that is the case. On the other hand, I can see how this being an option could be interpreted to _condone_ such changes as a general practice, which I would certainly like to avoid.

Given that concern, I think I'm leaning toward taking this out.